host-controller/src/main/java/org/jboss/as/host/controller/mgmt/HostControllerRegistrationHandler.java

```diff
@@ -699,8 +704,9 @@ boolean completeStep(Object result) {
             return setResult((T) result);
         }
 
-        boolean setFailed() {
-            return setFailed(null);
+        boolean failed(Throwable t) {
```

Comment:
Could you add a Java doc describing the argument cannot be null?

Reply:
Fixed.